Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Ride.hs

```diff
@@ -156,7 +156,7 @@ ticketRideList merchantShortId opCity mbRideShortId countryCode mbPhoneNumber _
     (Nothing, Just number) -> do
       no <- getDbHash number
       let code = fromMaybe "+91" countryCode
-      person <- runInReplica $ QPerson.findByMobileNumberAndMerchant code no merchant.id >>= fromMaybeM (PersonWithPhoneNotFound number)
+      person <- runInReplica $ QPerson.findByMobileNumberAndMerchantAndRole code no merchant.id DP.DRIVER >>= fromMaybeM (PersonWithPhoneNotFound number)
```

Comment:
check if re-indexing and re creating of unique constraints required after this.

Reply:
yes it  will required ideally. like this will ensure that the index reflects the new uniqueness constraints.

Reply:
Added Migration for that.